_draft-tests/SC3-1-1-html-lang-valid.md

```diff
@@ -0,0 +1,48 @@
+---
+rule_id: SC3-1-1-html-lang-valid
+test_case_template: default
+success_criterion:
+- 3.1.1
+---
+
+## Passed
+
+```html
+<html lang="fr">
+```
+
+```html
+<html xml:lang="fr">
+```
+
+```html
+<html xml:lang="fr" lang="fr">
+```
+
+```html
+<html lang="fr" xml:lang="xyz">
```

Comment:
This is identical to the last fail. The test case should be:

```html
<html lang="xyz" xml:lang="fr">
```

Reply:
Don't agree on this one. We are trying to depict a a pass case, where `lang` has the right value and `xml:lang` has an invalid value.

## Passed

```html
<html lang="fr">
```

```html
<html xml:lang="fr">
```

```html
<html xml:lang="fr" lang="fr">
```

```html
<html lang="fr" xml:lang="xyz">
```

## Failed

```html
<html lang="xyz">
```

```html
<html xml:lang="xyz">
```

```html
<html xml:lang="xyz" lang="fr">
```

Reply:
> If `lang` and `xml:lang` exists, then only `lang` is tested for conformance.

This means that the following would pass:

```html
<html xml:lang="xyz" lang="fr">
```

Reply:
You are right. I have edited the test case to:

```html
<html xml:lang="xyz" lang="xyz">
```